test/test_alexa_app_creation.js

```diff
@@ -0,0 +1,30 @@
+/*jshint expr: true*/
+"use strict";
+var chai = require("chai");
+var chaiAsPromised = require("chai-as-promised");
+chai.use(chaiAsPromised);
+var expect = chai.expect;
+chai.config.includeStack = true;
+
+describe("Alexa", function () {
+    var Alexa = require("../index");
```

Comment:
Looks like tabs. We've standardized all the things on 2 spaces, if you could please change that.